UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -84,125 +84,124 @@ public class EnvironmentConfiguration
     /// The mode is determined by the presence or absence of a Communicator. In
     /// the presence of a communicator, the academy is run in training mode where
     /// the states and observations of each agent are sent through the
-    /// communicator. In the absence of a communciator, the academy is run in
+    /// communicator. In the absence of a communicator, the academy is run in
     /// inference mode where the agent behavior is determined by the brain
     /// attached to it (which may be internal, heuristic or player).
     /// </remarks>
     [HelpURL("https://github.com/Unity-Technologies/ml-agents/blob/master/" +
-             "docs/Learning-Environment-Design-Academy.md")]
+        "docs/Learning-Environment-Design-Academy.md")]
     public abstract class Academy : MonoBehaviour
     {
         [SerializeField]
         public BroadcastHub broadcastHub = new BroadcastHub();
 
-        private const string kApiVersion = "API-9";
+        private const string k_KApiVersion = "API-9";
 
         /// Temporary storage for global gravity value
-        /// Used to restore oringal value when deriving Academy modifies it
-        private Vector3 originalGravity;
+        /// Used to restore original value when deriving Academy modifies it
+        private Vector3 m_OriginalGravity;
 
         /// Temporary storage for global fixedDeltaTime value
-        /// Used to restore oringal value when deriving Academy modifies it
-        private float originalFixedDeltaTime;
+        /// Used to restore original value when deriving Academy modifies it
+        private float m_OriginalFixedDeltaTime;
 
         /// Temporary storage for global maximumDeltaTime value
-        /// Used to restore oringal value when deriving Academy modifies it
-        private float originalMaximumDeltaTime;
+        /// Used to restore original value when deriving Academy modifies it
+        private float m_OriginalMaximumDeltaTime;
 
         // Fields provided in the Inspector
 
+        [FormerlySerializedAs("maxSteps")]
         [SerializeField]
         [Tooltip("Total number of steps per global episode.\nNon-positive " +
-                 "values correspond to episodes without a maximum number of \n" +
-                 "steps. Once the step counter reaches this maximum value, the " +
-                 "environment will reset.")]
-        int maxSteps;
+            "values correspond to episodes without a maximum number of \n" +
+            "steps. Once the step counter reaches this maximum value, the " +
+            "environment will reset.")]
+        int m_MaxSteps;
```

Comment:
Style guide indicates that these shouldn't be prefixed, right? (I didn't really care for the examples, but feels like we should get the public facing classes correct, and this could break for users on upgrade)

Reply:
This is a private variable, so it should be.